util/rand_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package util
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestRandString(t *testing.T) {
+	s, err := RandomID("prefix=")
+	t.Logf("Rand string: %s", s)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !strings.HasPrefix(s, "prefix=") {
+		t.Errorf("Invalid prefix: %s", s)
+	}
+	if len(s)-len("prefix.") != 52 {
```

Comment:
even with the above fix won't this fail sometimes? sometimes the base32 encoding will have padding / won't (or is that not true). It seems like if it does sometimes and doesn't sometimes, then the length of the random string (ignoring prefix) will be variable length (cause we strip it sometimes) so it won't always be 52 long. like sometimes it'll be 51 and still be working fine